packages/release-mastermind/types/index.d.ts

```diff
@@ -3,0 +1,265 @@
+import {
+  Condition,
+  IssueCondition,
+  PRCondition,
+  ProjectCondition,
+  ScheduleCondition
+} from '../src/conditions'
```

Comment:
Codacy has a fix for the issue: [Replace `'../src/conditions'` with `"../src/conditions";`](https://app.codacy.com/gh/Videndum/action-masterminds/pullRequest?prid=7872005)
```suggestion
} from "../src/conditions";
```